### PR TITLE
8252964: [vectorIntrinsics] jdk/incubator/vector/Short64VectorTests.java crashes on AVX512 machines

### PR DESCRIPTION
**Pull request by DamonFool**
JBS: https://bugs.openjdk.java.net/browse/JDK-8252964
The initial review thread: https://mail.openjdk.java.net/pipermail/panama-dev/2020-September/010532.html

Thanks.
Best regards,
Jie

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8252964](https://bugs.openjdk.java.net/browse/JDK-8252964): [vectorIntrinsics] jdk/incubator/vector/Short64VectorTests.java crashes on AVX512 machines

### Reviewers
 * [Sandhya Viswanathan](https://openjdk.java.net/census#sviswanathan) (@sviswa7 - Committer)

### Download
`$ git fetch https://git.openjdk.java.net/panama-vector pull/1/head:pull/1`
`$ git checkout pull/1`

### CODE REVIEW

**Review by sviswa7 — APPROVED** [review]

**Comment by DamonFool** [comment]
Thanks @sviswa7 for your review.

Is it OK to be pushed?
Or shall we wait for one more review? @PaulSandoz

**Comment by sviswa7** [comment]
@DamonFool It is ok to push.

**Pull request closed without merging**
